config/src/config/storage_config.rs

```diff
@@ -8,6 +8,33 @@ use std::{
     path::PathBuf,
 };
 
+#[derive(Copy, Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct RocksdbConfig {
+    /// Port selected RocksDB options for tuning underlying rocksdb instance.
+    /// see https://github.com/facebook/rocksdb/blob/master/include/rocksdb/options.h
+    /// for detailed explanations.
+    pub create_if_missing: bool,
+    pub create_if_missing_column_families: bool,
```

Comment:
`open()`, `open_readonly()` and `open_as_secondary()` use different values for these options, I don't think we should pass them in?